account_statement_bankaccount_completion/tests/test_bankaccount_completion.py

```diff
@@ -20,55 +20,52 @@
 #
 #
 from openerp.tests import common
-import time
+from openerp import fields, tools
+from openerp.modules import get_module_resource
 
 ACC_NUMBER = " BE38 7330 4038 5372 "
 
 
-class bankaccount_completion(common.TransactionCase):
+class BankAccountCompletion(common.TransactionCase):
 
     def setUp(self):
-        super(bankaccount_completion, self).setUp()
+        super(BankAccountCompletion, self).setUp()
+        tools.convert_file(self.cr, 'account',
+                           get_module_resource('account', 'test',
+                                               'account_minimal_test.xml'),
+                           {}, 'init', False, 'test')
+        self.account_move_obj = self.env["account.move"]
+        self.account_move_line_obj = \
+            self.env["account.move.line"]
```

Comment:
As you call it every time with a context, you might want to prepare this context here

As you use multiple time the same `with_context`, you can do something like:

   self.move_line_obj = self.env['account.move.line].with_context(...)